release_notes/ocp-4-15-release-notes.adoc

```diff
@@ -2681,6 +2681,8 @@ include::snippets/opm-rhel8-known-issue.adoc[]
 //[id="ocp-telco-ran-4-15-known-issues"]
 //
 
+* There is a known issue in this release preventing the creation of web terminals when logged into the cluster as `kubeadmin`. The terminal will return the message: `Error Loading OpenShift command line terminal: User is not a owner of the requested workspace.` This issue will be fixed in a future {product-title} release. (link:https://issues.redhat.com/browse/WTO-262[*WTO-262*])
```

Comment:
```suggestion
* There is a known issue in this release preventing the creation of web terminals when logged into the cluster as `kubeadmin`. The terminal will return the message: `Error Loading OpenShift command line terminal: User is not a owner of the requested workspace.` This issue will be fixed in a future {product-title} release. (link:https://issues.redhat.com/browse/WTO-262[*WTO-262*])
```
```suggestion
* There is a known issue in this release preventing the creation of web terminals when logged into the cluster as `kubeadmin`. The terminal will return the message: `Error Loading OpenShift command line terminal: User is not an owner of the requested workspace.` This issue will be fixed in a future {product-title} release. (link:https://issues.redhat.com/browse/WTO-262[*WTO-262*])
```
If this is the actual output I guess it can't be fixed.

Reply:
It is actual output